### PR TITLE
Use tag annotation if present when sending notifications

### PR DESCRIPTION
**Pull request by rwestberg**
Hi all,

Please review this change that updates the notifier to include the tag annotation if it is present when creating a new tag notification email. Also ensure that notifications are sent even if a tag does not conform to the OpenJDK tag parser format.

Best regards,
Robin

## Progress
- [x] Change must not contain extraneous whitespace
- [x] Change must be properly reviewed

## Approvers
 * Erik Helin ([ehelin](@edvbld) - **Reviewer**) **Note!** Review applies to fe09eede152c673f917315708b385ecb873e259b

### CODE REVIEW

File: bots/notify/src/main/java/org/openjdk/skara/bots/notify/JNotifyBot.java [hunk 1]
```diff
@@ -170,29 +170,27 @@ private void handleTags(Repository localRepo, UpdateHistory history) throws IOEx
                                 .filter(Optional::isPresent)
                                 .map(Optional::get)
                                 .sorted(Comparator.comparingInt(OpenJDKTag::buildNum))
-                                .map(OpenJDKTag::tag);
+                                .collect(Collectors.toList());
+
         var newNonJdkTags = newTags.stream()
-                                   .filter(tag -> OpenJDKTag.create(tag).isEmpty());
+                                   .filter(tag -> OpenJDKTag.create(tag).isEmpty())
+                                   .collect(Collectors.toList());
```
Comment (edvbld) [inline bots/notify/src/main/java/org/openjdk/skara/bots/notify/JNotifyBot.java]: Please move this below the for loop so it is next to the for loop of `newNonJdkTags`

Comment (rwestberg) [reply bots/notify/src/main/java/org/openjdk/skara/bots/notify/JNotifyBot.java]: Sure!

**Comment by kevinrushforth** [comment]
I look forwarded to testing this once you've updated the bot with this change.

**Pull request closed without merging**